Shuttle-back/src/main/java/com/shuttle/driver/DriverService.java

```diff
@@ -31,6 +41,50 @@ public Driver setAvailable(Driver driver, boolean available) {
 		
 		return driver;
 	}
+
+	@Override
+	public List<LocationDTO> getActiveDriversLocations() {
+		List<Driver> activeDrivers = driverRepository.findByAvailableTrue();
+		List<Vehicle> driversVehicles = vehicleRepository.findByDriverIn(activeDrivers);
```

Comment:
`DriverService` should *only* ever communicate with `VehicleService`. Create methods `findByDriverIn()` in `VehicleService` and call that here.